packages/react-native-editor/src/setup-locale.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { setLocaleData } from '@wordpress/i18n';
+
+export default ( { locale, extraTranslations, domain, getTranslation } ) => {
```

Comment:
This function sets the locale data for a domain, when the domain is not set it will use the default domain ([reference 1](https://github.com/WordPress/gutenberg/blob/trunk/packages/i18n/README.md#setlocaledata), [reference 2](https://github.com/WordPress/gutenberg/blob/d2f0c66c3e10e5c0c7aa7f1480506a6d0c1e3ba6/packages/i18n/src/create-i18n.js#L174)).